<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_18" >
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:13pt; font-weight:600;">NX Client&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_15" >
            <property name="text" >
             <string>Server hostname:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="server" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_16" >
            <property name="text" >
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="username" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_17" >
            <property name="text" >
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="password" >
            <property name="inputMask" >
             <string/>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="echoMode" >
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Session Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="session_name" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Session Type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="session_type" >
            <item>
             <property name="text" >
              <string>unix-kde</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>unix-gnome</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>unix-application</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>windows</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>vnc</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Memory cache size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cache" >
            <item>
             <property name="text" >
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>32</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Disk cache size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="images" >
            <item>
             <property name="text" >
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>32</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Link type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="link_type" >
            <item>
             <property name="text" >
              <string>modem</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>isdn</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>adsl</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>wan</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>lan</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Resolution:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="resolution" >
            <item>
             <property name="text" >
              <string>640x480</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>800x600</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>1024x768</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>1600x1200</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>RENDER Extension:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="render" >
            <property name="text" >
             <string>Use RENDER</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Use SSH Tunnelling:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="encryption" >
            <property name="text" >
             <string>Use SSL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>Image compression:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="image_compression" >
            <item>
             <property name="text" >
              <string>JPEG</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>RAW</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>PNG</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_11" >
            <property name="text" >
             <string>JPEG Quality:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QSlider" name="jpeg_quality" >
              <property name="maximum" >
               <number>100</number>
              </property>
              <property name="sliderPosition" >
               <number>75</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>75</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_12" >
            <property name="text" >
             <string>RDP Username:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rdp_username" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_13" >
            <property name="text" >
             <string>RDP Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rdp_password" >
            <property name="echoMode" >
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_14" >
            <property name="text" >
             <string>RDP Server:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rdp_server" />
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton" >
        <property name="text" >
         <string>&amp;Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit" />
   </widget>
   <addaction name="menu_File" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
